device/ingrasys/x86_64-ingrasys_s8900_64xc-r0/INGRASYS-S8900-64XC/port_config.ini

```diff
@@ -0,0 +1,65 @@
+# name          lanes             alias
+Ethernet0       17                 Ethernet0 
+Ethernet1       18                 Ethernet1 
+Ethernet2       19                 Ethernet2 
+Ethernet3       20                 Ethernet3 
+Ethernet4       24                 Ethernet4 
+Ethernet5       23                 Ethernet5 
+Ethernet6       22                 Ethernet6 
+Ethernet7       21                 Ethernet7 
+Ethernet8       28                 Ethernet8 
+Ethernet9       27                 Ethernet9 
+Ethernet10      26                 Ethernet10
+Ethernet11      25                 Ethernet11
+Ethernet12      32                 Ethernet12
+Ethernet13      31                 Ethernet13
+Ethernet14      30                 Ethernet14
+Ethernet15      29                 Ethernet15
+Ethernet16      1                  Ethernet16
+Ethernet17      2                  Ethernet17
+Ethernet18      3                  Ethernet18
+Ethernet19      4                  Ethernet19
+Ethernet20      8                  Ethernet20
+Ethernet21      7                  Ethernet21
+Ethernet22      6                  Ethernet22
+Ethernet23      5                  Ethernet23
+Ethernet24      12                 Ethernet24
+Ethernet25      11                 Ethernet25
+Ethernet26      10                 Ethernet26
+Ethernet27      9                  Ethernet27
+Ethernet28      13                 Ethernet28
+Ethernet29      14                 Ethernet29
+Ethernet30      15                 Ethernet30
+Ethernet31      16                 Ethernet31
+Ethernet32      33                 Ethernet32
+Ethernet33      34                 Ethernet33
+Ethernet34      35                 Ethernet34
+Ethernet35      36                 Ethernet35
+Ethernet36      38                 Ethernet36
+Ethernet37      39                 Ethernet37
+Ethernet38      40                 Ethernet38
+Ethernet39      37                 Ethernet39
+Ethernet40      41                 Ethernet40
+Ethernet41      42                 Ethernet41
+Ethernet42      43                 Ethernet42
+Ethernet43      44                 Ethernet43
+Ethernet44      48                 Ethernet44
+Ethernet45      45                 Ethernet45
+Ethernet46      46                 Ethernet46
+Ethernet47      47                 Ethernet47
```

Comment:
As with the file above, the `alias` column here is optional, and seeing as your aliases are exactly the same as their corresponding SONiC port names, the aliases serve no purpose for this device. You can safely remove the `alias` column in this file.